src/style.css

```diff
@@ -2,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+.search-bar__input {
+  width: 100%;
+  display: flex;
+  border: 1px solid rgba(225, 232, 237, 1);
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  color: rgba(61, 78, 97, 1);
+  box-shadow: 0 1px 8px #3d4e611a;
+  font-style: normal;
+  border-radius: 4px;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+  font-size: 16px;
+  padding-left: 62px;
+  margin-top: 20px;
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-bar__input--big:hover {
+  box-shadow: 0 4px 4px #00000040;
+}
+
+.search-bar__input--big:focus {
+  font-weight: 400;
+  background-color: #f6f6f7;
+}
+
+.search-bar__label--big::after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  color: rgba(104, 124, 148, 1);
+}
+
+.search-bar__label--small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  color: rgba(104, 124, 148, 1);
+}
+
+.search-bar__input--small {
+  height: 42px;
+  background-position: 13px 14px;
+  background-size: 11px 11px;
+  font-size: 14px;
+  padding-left: 33px;
+  margin-top: 20px;
+  color: rgba(61, 78, 97, 1);
+}
+
+.search-bar__input--small:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+  font-weight: 400;
```

Comment:
The checklist advises being consistent with your margins by adding only top or bottom, but not both. Consider reviewing the margin usage to ensure consistency.